2025-07-24 14:29:44,098 - root - logging_q2.py - ERROR - Line: 25 - ValueError. Enter an integer.
